enterprise_access/apps/api/v1/views.py

```diff
@@ -826,7 +828,11 @@ def redeem(self, request, *args, **kwargs):
                 )
             if policy.can_redeem(learner_id, content_key):
                 response = policy.redeem(learner_id, content_key)
-
+                if settings.KAFKA_ENABLED:  # pragma: no cover
```

Comment:
I think we should move the `if` condition inside the `send_subsidy_redemption_event_to_event_bus`. The caller of this method should not take the responsibility to check if `Kafka` is enabled/disabled.

Reply:
good point ... done